### PR TITLE
Add new sort of well and dl-horizontal-item

### PR DESCRIPTION
**Pull request by Tankenstein**

### CODE REVIEW

File: less/wells.less [hunk 1]
```diff
@@ -2,22 +2,39 @@
...
+  @media only screen and (min-width: @size) {
+    .well();
+  }
 }
 
 // Sizes
-.well-lg {
-  padding: 24px;
-  border-radius: @border-radius-large;
+.well-xs {
```
Comment (StevePole) [inline less/wells.less]: I think you need to add `.well, .well-xs` for backwards compatibility

**Review by StevePole — CHANGES_REQUESTED** [review]
One thing, otherwise good.

File: less/wells.less (hunk 1 above)
Comment (Tankenstein) [reply less/wells.less]: Righto, i checked across our repos for 0 usages but will do it just in case

Comment (Tankenstein) [reply less/wells.less]: Espec. since i don't want to increment version major. Done though.

**Review by StevePole — APPROVED** [review]

**Review by oliverviljamaa — APPROVED** [review]

**Pull request merged**